src/chordsymbol.ts

```diff
@@ -490,11 +490,12 @@ export class ChordSymbol extends Modifier {
   // ChordSymbol allows multiple blocks so we can mix glyphs and font text.
   // Each block can have its own vertical orientation
   // eslint-disable-next-line
-  getSymbolBlock(parameters: any): any {
-    parameters = parameters == undefined ? {} : parameters;
-    const symbolType = parameters.symbolType ? parameters.symbolType : ChordSymbol.symbolTypes.TEXT;
-    const text = parameters.text ? parameters.text : '';
-    const symbolModifier = parameters.symbolModifier ? parameters.symbolModifier : ChordSymbol.symbolModifiers.NONE;
+  getSymbolBlock(parameters?: any): any {
```

Comment:
Why not just `getSymbolBlock(parameters = {})` and remove the next line?

Reply:
In order to still allow the usage of `null`.

Reply:
I'm happy with either solution. The inline default argument `parameters={}` looks clean. This approach is also used in other places in VexFlow.

However, it is true that the nullish coalescing approach is more robust. If there is older code out there that passes in a null, it will still work as expected.

If we don't care about the extra robustness / supporting `null` (since this is a 4.0.0 release with breaking changes anyways), we could just stick with the inline default argument approach.

Reply:
Yeah, I think not supporting `null` is fine here.

Reply:
OK, I'll go for the inline (parameters = {}) approach. It's also used elsewhere in VexFlow.